src/openrct2/CmdlineSprite.cpp

```diff
@@ -598,6 +598,13 @@ int32_t cmdline_for_sprite(const char** argv, int32_t argc)
 
         json_t sprite_description;
 
+        auto spriteFile = SpriteFile::Open(spriteFilePath);
+        if (!spriteFile.has_value())
```

Comment:
There isn't any point opening this file. It was created on line 595. Just skip the save/load.

Reply:
ah I see, I missed that save call; that's also why its breaking an existing file even on failure since it instantly overwrites it with just the header